aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -0,0 +1,427 @@
+    },
+    "ProcessorFeatures": {
+      "type": "array",
+      "uniqueItems": true,
+      "items": {
+        "$ref": "#/definitions/ProcessorFeature"
+      },
+      "description": "The number of CPU cores and the number of threads per core for the DB instance class of the DB instance."
+    },
+    "PromotionTier": {
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 15,
+      "default": 1,
+      "description": "A value that specifies the order in which an Aurora Replica is promoted to the primary instance after a failure of the existing primary instance."
+    },
+    "PubliclyAccessible": {
+      "type": "boolean",
+      "description": "Indicates whether the DB instance is an internet-facing instance. If you specify true, AWS CloudFormation creates an instance with a publicly resolvable DNS name, which resolves to a public IP address. If you specify false, AWS CloudFormation creates an internal instance with a DNS name that resolves to a private IP address."
+    },
+    "SourceDBInstanceIdentifier": {
+      "type": "string",
+      "description": "If you want to create a Read Replica DB instance, specify the ID of the source DB instance. Each DB instance can have a limited number of Read Replicas."
+    },
+    "SourceRegion": {
+      "type": "string",
+      "description": "The ID of the region that contains the source DB instance for the Read Replica."
+    },
+    "StorageEncrypted": {
+      "type": "boolean",
+      "description": "A value that indicates whether the DB instance is encrypted. By default, it isn't encrypted."
+    },
+    "StorageType": {
+      "type": "string",
+      "enum": [
+        "standard",
+        "gp2",
+        "io1"
+      ],
+      "description": "Specifies the storage type to be associated with the DB instance."
+    },
+    "Tags": {
+      "type": "array",
+      "uniqueItems": true,
+      "items": {
+        "$ref": "#/definitions/Tag"
+      },
+      "description": "Tags to assign to the DB instance."
+    },
+    "Timezone": {
+      "type": "string",
+      "description": "The time zone of the DB instance. The time zone parameter is currently supported only by Microsoft SQL Server."
+    },
+    "UseDefaultProcessorFeatures": {
+      "type": "boolean",
+      "description": "A value that indicates whether the DB instance class of the DB instance uses its default processor features."
+    },
+    "VPCSecurityGroups": {
+      "type": "array",
+      "uniqueItems": true,
+      "items": {
+        "type": "string"
+      },
+      "description": "A list of the VPC security group IDs to assign to the DB instance. The list can include both the physical IDs of existing VPC security groups and references to AWS::EC2::SecurityGroup resources created in the template."
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "DBInstanceClass"
+  ],
+  "createOnlyProperties": [
```

Comment:
you should specify `conditionalCreateOnlyProperties` - [Adding conditionallyCreateOnlyProperties to metaschema (#121)](https://github.com/aws-cloudformation/cloudformation-resource-schema/commit/7d82244d54d4178b9a0f5688c39d1050b0ff3cb4)

```
"conditionalCreateOnlyProperties": [
  "/properties/AutoMinorVersionUpgrade",
  "/properties/BackupRetentionPeriod",
  "/properties/DBParameterGroupName",
  "/properties/Engine",
  "/properties/EngineVersion",
  "/properties/MonitoringInterval",
  "/properties/PerformanceInsightsKMSKeyId",
  "/properties/PreferredMaintenanceWindow",
  "/properties/StorageType"
]

```

Reply:
Added. Thank you.